pkgs/tools/filesystems/dduper/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchFromGitHub, btrfs-progs, python3 }:
+
+let
+  btrfsProgsPatched = btrfs-progs.overrideAttrs (oldAttrs: {
+    version = "5.9";
+    patches =
+      [ ./0001-Print-csum-for-a-given-file-on-stdout.patch
```

Comment:
can you fetch this patch with fetchpatch?
Or alternatively, explain in a comment how you obtained it?

Reply:
It's from the master branch of the `dduper` GitHub repository. Should I add a comment stating this, or convert to `fetchPatch`?

Reply:
nixpkgs prefers fetchpatch if possible.